lib/broadside.rb

```diff
@@ -25,21 +24,15 @@ def self.configure
   end
 
   def self.load_config(config_file)
-    begin
-      load USER_CONFIG_FILE if File.exists?(USER_CONFIG_FILE)
-    rescue LoadError => e
-      error "Encountered an error loading system configuration file '#{USER_CONFIG_FILE}' !"
-      raise e
-    end
+    raise ArgumentError, "#{config_file} does not exist" unless File.exist?(config_file)
 
+    config.config_file = config_file
     begin
-      config.config_file = config_file
-      load config_file
-    rescue LoadError => e
-      error "Encountered an error loading required configuration file '#{config_file}' !"
-      raise e
+      [config_file, USER_CONFIG_FILE].each { |file| load file if File.exist?(file) }
```

Comment:
(i now officially feel guilty for deleting the spec that checks load order - when i was unifying all the spec contexts that particular spec caused some difficulties and i thought [these lines would force a call order](https://github.com/lumoslabs/broadside/blob/master/spec/broadside_spec.rb#L17-L18) but i just checked and they do not.  😞 )